db/schema.rb

```diff
@@ -243,7 +243,7 @@
     t.integer  "income"
     t.boolean  "refund",                        null: false
     t.date     "date_fee_paid"
-    t.boolean  "probate"
+    t.boolean  "probate",                       null: false
```

Comment:
is this going to cause any issues?

Reply:
Hmm, I never noticed that - I never changed it - my guess is that the schema.rb in master is wrong as I built this db from scratch.  Ill double check what migrations say

Reply:
According to the migrations, this is correct - see db/migrate/20160308155249_add_online_application_tables.rb

Reply:
yes, that looks to be the case. Probably worth moving it to a separated commit?